### ISSUE TITLE
Split dataset

### ISSUE THREAD

**Issue by cla-cif** [cla-cif/Cherry-Powdery-Mildew-Detector#60]
Related to #17

**Comment by cla-cif** [comment]
- The training set is divided into a 0.70 ratio of data.
- The validation set is divided into a 0.10 ratio of data.
- The test set is divided into a 0.20 ratio of data.
